src/react/index.js

```diff
@@ -16,7 +16,7 @@ export {
 
 const push = (obj, items) => Array.prototype.push.apply(obj, items)
 
-export default function (tag, objs, vars = [], content) {
+export default function (tag, cls, objs, vars = [], content) {
```

Comment:
We could but in the future I think we should remove `objs` and move all the things from composes into content